sr_edc_launch/sr_edc_bimanual_ros_control.launch

```diff
@@ -2,6 +2,9 @@
   <arg name="debug" default="false"/>
 
   <!-- ROBOT CONFIG-->
+  <!-- setting this parameter to false allows to load the robot_description from a higher level -->
+  <arg name="load_robot_description_at_lower_level" default="true"/>
```

Comment:
Is this the "lowest" level? If so, do you think we should rename this argument?

Reply:
makes sense, should just be load_robot_description